src/pkg/bundle/deploy.go

```diff
@@ -57,70 +55,23 @@ func (b *Bundle) Deploy() error {
 		}
 	}
 
-	// get bundle state
-	var sc *state.Client
-	var kc *cluster.Cluster
-	if config.FF_STATE_ENABLED {
-		message.Debugf("state management disabled, skipping bundle state management")
-		var err error
-		var enabledState bool
-		kc, err = cluster.NewCluster()
-		if err != nil {
-			// common scenario for Zarf actions run before cluster is available
-			enabledState = false
-		}
-		sc, err = state.NewClient(kc, enabledState)
-		if err != nil {
-			return err
-		}
-
-		err = sc.InitBundleState(&b.bundle, state.Deploying)
-		if err != nil {
-			return err
-		}
-	} else {
-		sc = &state.Client{
-			Enabled: false,
-		}
-	}
-
 	// if resume, filter for packages not yet deployed
 	if b.cfg.DeployOpts.Resume {
-		deployedPackageNames := state.GetDeployedPackageNames()
-		var notDeployed []types.Package
+		deployedPackageNames := GetDeployedPackageNames()
+		notDeployed := []types.Package{}
```

Comment:
Curious about `var notDeployed []types.Package` vs `notDeployed := []types.Package{}` I understand there are subleties, but do you prefer one over the other?

Reply:
I typically prefer `var` style over `:= {}` myself, unless at the time I already have elements I want to start with. In this case I believe I was just reverting the file to it's pervious version, but maybe I made that change inadvertently. Either way, I changed it to the `var` style in c7b8ec8